examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>where is <code>hvd.init()</code> command? is it happening under the hood? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Yes it happens under the hood</p>